demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -96,6 +96,7 @@ def is_valid_scheme(self):
         if self.scheme_name in [None, 'image']:
             return True
         try:
+            logging.disable(logging.INFO)
```

Comment:
please add a comment what are we doing here

Reply:
added